integration/swarm/swarm.bats

```diff
@@ -90,6 +90,21 @@ setup() {
 	export http_proxy="$proxy"
 }
 
+@test "check mtu values in different interfaces" {
+	REPLICAS_UP=$($DOCKER_EXE ps -q --filter name="${SERVICE_NAME}")
```

Comment:
Just a couple of thoughts:
- we repeat the `REPLICAS_UP` idiom through this file - i wonder if that can be lifted out/re-factored?
- we could check that REPLICAS_UP is not empty here (and maybe other places) as a sanity check.